sycl/test-e2e/AddressSanitizer/invalid-argument/bad-context.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: linux, (gpu && level_zero), cpu
+// REQUIRES: linux, (gpu && level_zero), any-device-is-cpu
```

Comment:
```suggestion
// REQUIRES: linux
// UNSUPPORTED: (opencl && gpu), acc
```

I general I have a very same question about how switching to `any-device-is-cpu` helps here